### ISSUE TITLE
Differences between profiles

### ISSUE THREAD

**Issue by LoiLock** [sclebo05/KobraNeoProfiles#5]
I noticed there's a nozzle temperature difference between Cura (205c) and Prusa (201c). Is there a reason for this that I'm missing?

Prusa also has separate files for filament settings, which are present for Cura. Do you want me to create a PR to get these to match up?

The Cura profile works really well, but I'd like to give Prusa a shot since, from the looks of things, there's more options for to improve bed adhesion

**Comment by sclebo05** [comment]
There is no significant reason for the temperature change - I probably did a check-in with a different filament.  I tend to print my PLA+ a little hotter, and cheap PLA at 201. Temperature is one area that everyone should test for themselves.

I'd welcome any changes or additions. I'm always learning myself, and frankly I've reached a point where prints in either slicer work 'good enough' for me.  At some point, I may submit my profiles to the slicer projects for possible inclusion in future versions.

Glad the Cura profile works well for you, I started there.  I've since moved to Prusa as I like the workflow uploading to Octoprint a bit better.

**Comment by LoiLock** [comment]
I personally haven't had any luck moving the filament settings from Cura over to Prusa, since it seems they're spread out. Could you share the filament and print settings profiles? I want to move over to Prusa for the same reason

**Comment by sclebo05** [comment]
For filament I'm using 'Generic PLA' included in Cura.  

As for migrating settings, I created the initial profile in Cura, then used a dual-monitor setup and painstakingly mirrored settings between the two apps to create my Prusa profile.  If anyone knows of a migration tool, I'll gladly try it.

I actually stepped away from Cura for the most part, I dislike that the .curaprofile file format isn't directly editable. I live in Prusa for my day to day printing now.
